website/docs/r/resource_provider_registration.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+subcategory: "Base"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_resource_provider_registration"
+description: |-
+    Registers a resource provider.
+---
+
+# azurerm_resource_provider_registration
+
+Registers a resource provider.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_provider_registration" "example" {
+  resource_provider_namespace = "Microsoft.PolicyInsights"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `resource_provider_namespace` - (Required) The namespace of the resource provider.
+
+## Timeouts
+
+The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) for certain actions:
+
+* `create` - (Defaults to 30 minutes) Used when registering the Resource Provider Namespace.
+* `update` - (Defaults to 30 minutes) Used when registering the Resource Provider Namespace.
+* `read` - (Defaults to 5 minutes) Used when retrieving the Resource Provider Namespace.
+* `delete` - (Defaults to 30 minutes) Used when unregistering the Resource Provider Namespace.
```

Comment:
this resource also supports Import - can we include an example of how to do that ([example](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/resource_group.html.markdown#L49-L55))